ctonly/ct.go

```diff
@@ -84,6 +84,28 @@ func (c Entry) LeafData(idx uint64) []byte {
 	return b.BytesOrPanic()
 }
 
+// MerkleTreeLeaf returns a RFC 6962 MerkleTreeLeaf.
+func (e *Entry) MerkleTreeLeaf(idx uint64) []byte {
```

Comment:
optional: I think `MerkleLeafHash` below is identical to this, modulo a call to `HashLeaf`, so I think we could refactor that to just call this method before hashing.

Feel free to leave that for another PR (or for me to do), just noting it.

Reply:
Done!